File: src/main/java/ru/vaschenko/ParkPoint/model/Photo.java
package ru.vaschenko.ParkPoint.model;

import jakarta.persistence.*;

import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
@Entity
@Table(name = "Photo")
public class Photo {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @ManyToOne
    @JoinColumn(name = "id_parking_zone")
    private ParkingZone parkingZone;

    private Integer order;
    private String photoUrl;

}

File: src/dto/Photo.js
export class Photo {
    constructor(id, parkingZone, order, photoUrl) {
      this.id = id;
      this.parkingZone = parkingZone; // Объект ParkingZoneDto
      this.order = order;
      this.photoUrl = photoUrl;
    }
  }
  